graphql-dgs-platform/build.gradle.kts

```diff
@@ -48,27 +48,28 @@ dependencies {
     constraints {
         // GraphQL Platform
         api("com.graphql-java:graphql-java") {
-            version { require("18.3") }
+            version { require("19.2") }
         }
         api("com.graphql-java:graphql-java-extended-scalars") {
-            version { require("18.1") }
+            version { require("19.0") }
         }
         api("com.graphql-java:graphql-java-extended-validation") {
             // The version below will work with Jakarta EE 8 and use Hibernate Validator 6.2.
+            // It is meant to work on graphql-java 18.x, that said there is no version for it yet.
             version { require("18.1-hibernate-validator-6.2.0.Final") }
```

Comment:
There is no `19.0-hibernate-validator-6.2.0.Final` version, nor I think there will be.

Reply:
Thanks to @setchy for pointing out that there is a `v19.1-validator-6.2.0.Final`! Will upgrade.